src/Rocketlet.ts

```diff
@@ -9,14 +10,10 @@ export abstract class Rocketlet {
      * Also, please use the `initialize()` method to do items instead of the constructor as the constructor
      * *might* be called more than once but the `initialize()` will only be called once.
      */
-    protected constructor(private readonly name: string,
-                          private readonly id: number,
-                          private readonly version: string,
-                          private readonly description: string,
-                          private readonly requiredApiVersion: string,
-                          private readonly author: IRocketletAuthor) {
-        console.log(`Constructed the Rocketlet ${this.name} (${this.id})`,
-            `v${this.version} which depends on the API v${requiredApiVersion}! Created by ${author.name}`);
+    protected constructor(private readonly info: IRocketletInfo) {
```

Comment:
`IRocketletInfo` will be passed into the Rocketlet's constructor which is parsed from the included `rocketlet.json` file.